loops.js

```diff
@@ -50,4 +63,12 @@
 // >> 8
 // >> 9
 // ==========================================
-
+for(let i =0; i < 10; i++) {
+    if(i >= 3 && i <= 5) {
+        console.log("> " + i);
+    }else if(i >= 6) {
+        console.log(">> " + i);
+    } else {
+        console.log(i);
+    }
```

Comment:
Top! 

Ook hier staat in de uitwerkingen nog een andere manier, mocht je interesse hebben: https://github.com/hogeschoolnovi/frontend-javascript-loops-uitwerkingen/blob/uitwerkingen/loops.js